src/Game.cpp

```diff
@@ -19,3 +19,8 @@ int Game::convertSingleCharToPoints(char input)
 
 void Game::read (const std::string &) {}
 
+void readFrames (std::string frames){
+    int placeOfPipe = frames.find('|');
+    frames.append(frames.begin() + placeOfPipe+1, frames.end());
```

Comment:
Nie jest to poprawna implementacja, ale jeszcze tego nie wiesz, bo nie masz testów :dagger: 
Powinnaś szukać dwukropka, ale wziąć wszystko od niego do końca.